travis/build.sh

```diff
@@ -2,12 +2,12 @@
 set -ex
 # setup config file for integration tests
 cp BridgeAdminCredentials.plist ../BridgeAdminCredentials.plist
-if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then     # on pull requests
-    bundle exec fastlane doc scheme:"BridgeSDK"
+# build docs
+bundle exec fastlane doc scheme:"BridgeSDK"
```

Comment:
Is this expecting the BridgeSDK scheme to build docs? (because it doesn't as far as I can see) or is it telling fastlane to build the doc scheme (which it somehow knows is "Documentation") in the BridgeSDK Xcode project?

Reply:
yes, this is the fastlane command to build the doc lane which is defined in fastlane/Fastfile